robot/EDA/resources/ContactsPageObject.py

```diff
@@ -457,9 +464,20 @@ def Verify_contact_values(self, FirstName, LastName):
             "Details tab not found on contact", 
             False
         )
+#        self.selenium.driver.refresh()
```

Comment:
Is this part of code commented intentionally?

Reply:
Yes

Reply:
Why? Can we just remove it?